src/contexts/Drive/Drive.context.tsx

```diff
@@ -46,6 +48,52 @@ interface DriveContextProviderProps {
 const logger = new BaseLogger({
   tag: 'DRIVE_CONTEXT',
 });
+
+export const removeOldItemsFromLocalDB = async () => {
+  const lastUpdatedAt = (await asyncStorageService.getItem(AsyncStorageKey.LastUpdatedAt)) || new Date().toISOString();
+
+  const [modifiedFiles, modifiedFolders] = await Promise.all([
+    driveFileService.getModifiedFiles({
+      updatedAt: lastUpdatedAt,
```

Comment:
You need to save filesLastUpdatedAt and foldersLastUpdatedAt separatedly, as the most recent folder could be newer than the most recent file, and you will miss items

EDIT: Never mind, I saw below than only the most recent will be picked, cool solution